database/Seeders/eggs/rust/egg-rust.json

```diff
@@ -20,7 +20,7 @@
         "ghcr.io\/parkervcp\/games:rust": "ghcr.io\/parkervcp\/games:rust"
```

Comment:
This is not exported from a panel as the timestamp did not update

Reply:
No idea what you are talking about :x